pkg/kudoctl/kudoinit/crd/bindata.go

```diff
@@ -96,7 +96,7 @@ func configCrdsKudoDev_instancesYaml() (*asset, error) {
 		return nil, err
 	}
 
-	info := bindataFileInfo{name: "config/crds/kudo.dev_instances.yaml", size: 1040, mode: os.FileMode(420), modTime: time.Unix(1576836957, 0)}
+	info := bindataFileInfo{name: "config/crds/kudo.dev_instances.yaml", size: 1040, mode: os.FileMode(420), modTime: time.Unix(1577106046, 0)}
```

Comment:
these changes will go away once #1236 lands

Reply:
just out of curiosity: why do you need them at all?

Reply:
we don't and these cause headaches imo